apps/nextjs-website/react-components/components/Editorial/Image.tsx

```diff
@@ -27,8 +27,8 @@ export const Image = ({ image, mobileImage, pattern = 'none', breakpoint = 834 }
 
   const desktopStyle = {
     ...image.props.style,
-    objectFit: 'cover',
-    objectPosition: 'center',
+    objectFit: 'contain', 
+    objectPosition: 'center', 
```

Comment:
```suggestion
    objectFit: 'contain',
    objectPosition: 'center',
```
Random trailing spaces